frontend/src/Pages/AllUser/Register/Register.jsx

```
@@ -0,0 +1,89 @@
+        initial={{ opacity: 0, scale: 0.9 }}
+        animate={{ opacity: 1, scale: 1 }}
+        transition={{ duration: 0.5 }}
+        className="bg-white p-8 rounded-xl shadow-2xl w-full max-w-md">
+        <h2 className="text-center text-3xl font-bold text-indigo-600">Sign Up</h2>
+        <form onSubmit={handleSignUp} className="space-y-4 mt-6">
+          <div>
+            <label className="block text-sm font-medium text-gray-700">Name</label>
+            <input type="text" className="mt-1 p-2 w-full border rounded-md" value={name} onChange={(e) => setName(e.target.value)} required />
+          </div>
+          <div>
+            <label className="block text-sm font-medium text-gray-700">Email</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)